libsel4vm/src/boot.c

```diff
@@ -92,6 +92,15 @@ int vm_assign_vcpu_target(vm_vcpu_t *vcpu, int target_cpu)
         ZF_LOGE("Failed to assign target cpu - A VCPU is already assigned to core %d", target_cpu);
         return -1;
     }
+
+#if CONFIG_MAX_NUM_NODES > 1
+    int err = seL4_TCB_SetAffinity(vcpu->tcb.tcb.cptr, target_cpu);
```

Comment:
I think to commit is ok. But it makes this generic code now, how do other architectures cope with this change then? Do we gave to change something in the x86 VMM part also?

Reply:
I believe `vm_assign_vcpu_target` is currently only used by ARM